tfx/components/example_validator/executor_test.py

```diff
@@ -47,25 +47,43 @@ def testDo(self):
 
     validation_output = standard_artifacts.ExampleAnomalies()
     validation_output.uri = os.path.join(output_data_dir, 'output')
+    validation_output.split_names = artifact_utils.encode_split_names(
+        ['train', 'eval'])
 
     input_dict = {
         executor.STATISTICS_KEY: [eval_stats_artifact],
         executor.SCHEMA_KEY: [schema_artifact],
     }
+
+    exec_properties = {
+        executor.EXCLUDE_SPLITS_KEY: ['test']
+    }
+
     output_dict = {
         executor.ANOMALIES_KEY: [validation_output],
     }
 
-    exec_properties = {}
-
     example_validator_executor = executor.Executor()
     example_validator_executor.Do(input_dict, output_dict, exec_properties)
-    self.assertEqual(['anomalies.pbtxt'],
-                     tf.io.gfile.listdir(validation_output.uri))
-    anomalies = io_utils.parse_pbtxt_file(
-        os.path.join(validation_output.uri, 'anomalies.pbtxt'),
-        anomalies_pb2.Anomalies())
-    self.assertNotEqual(0, len(anomalies.anomaly_info))
+
+    # Check example_validator outputs.
+    train_anomalies_path = os.path.join(validation_output.uri, 'train',
+                                        'anomalies.pbtxt')
+    eval_anomalies_path = os.path.join(validation_output.uri, 'eval',
+                                       'anomalies.pbtxt')
+    self.assertTrue(tf.io.gfile.exists(train_anomalies_path))
+    self.assertTrue(tf.io.gfile.exists(eval_anomalies_path))
+    train_anomalies = io_utils.parse_pbtxt_file(train_anomalies_path,
+                                                anomalies_pb2.Anomalies())
+    eval_anomalies = io_utils.parse_pbtxt_file(eval_anomalies_path,
+                                               anomalies_pb2.Anomalies())
+    self.assertEqual(0, len(train_anomalies.anomaly_info))
+    self.assertNotEqual(0, len(eval_anomalies.anomaly_info))
```

Comment:
schema under testdata might need to be updated, current schema is generated based on train, thus there isn't anomaly_info for train. since we change it to both train and eval, there shouldn't be anomaly_info for eval too